elements.go

```diff
@@ -30,3 +30,24 @@ type groupMembership struct {
 	XMLName xml.Name        `xml:"DAV: group-membership"`
 	Hrefs   []internal.Href `xml:"href"`
 }
+
+// ConditionalHeaderValue represents the value of a conditional header
+// according to RFC 2068 section 14.25 and RFC 2068 section 14.26
+// The (optional) value can either be a wildcard or an ETag.
+type ConditionalHeaderValue string
+
+func (val ConditionalHeaderValue) IsSet() bool {
+	return val != ""
+}
+
+func (val ConditionalHeaderValue) IsWildcard() bool {
+	return val == "*"
+}
+
+func (val ConditionalHeaderValue) ETag() (string, error) {
```

Comment:
Not related to this PR, but the RFC allows multiple ETags in here. Opened https://github.com/emersion/go-webdav/issues/117 to track this.